src/lib/spacefinder/article-body-adverts.ts

```diff
@@ -384,7 +384,7 @@ const attemptToAddInlineMerchAd = (): Promise<boolean> => {
 
 	const insertAds: SpacefinderWriter = (paras) =>
 		insertAdAtPara(
-			paras[0],
+			paras[0] as Node,
```

Comment:
Wonder if we can type the first parameter to `insertAdAtPara` as a HTMLElement to avoid this cast?

Reply:
The cast was to avoid dealing with undefined instead of there being a type conflict between HTMLElement and Node - happy to rethink how to do this without a cast though!